modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,27 @@
+[[hostname-rename]]
+= Rename {productname} Server
+
+It is possible to rename a {productname} Server.
+You first must change the network settings of the server locally and remotely.
+Then the [command]``spacewalk-hostname-rename`` script will the settings in the {productname} PostgreSQL database and the internal structures of {productname} accordingly.
+
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+
+
+
+.Procedure: Renaming {productname} Server
+. Change the network settings of the server on the system level locally and remotely at the DNS server.
+This part of the procedure is similar to the actions to take if you want to rename any other regular system.
++
+[NOTE]
+====
+You must also provide configuration settings for reverse name resolution.
+====
+. Restart your network configuration to make the {productname} server use the new hostname.
+If the server does not use the new name, the script will not complete.
+After rebooting the server will use the new name.
+. Run the script [command]``spacewalk-hostname-rename`` script with the public IP address of the server.
+. Some other actions are needed to make your landscape aware of the new hostname and IP address.
+Change all Salt clients to point to the new master, and with a re-activation key re-register the traditional clients (if there are any).
```

Comment:
Can probably drop some xrefs in here.

Reply:
Ah, wait here it is!

Then I am not sure what "other actions" could mean. An expert should advice :-)

Reply:
I am not sure if you need to re-onboard / re-register the clients. Would be good to try this out.

Reply:
Btw, I think if there are proxies that are being used for autoinstallations, it might be needed to reconfigure tftp-sync-recv (do not remember the right name from head, but I think you know what I mean) on the affected proxies as well.